pyanaconda/modules/payload/live/live_image.py

```diff
@@ -121,3 +144,135 @@ def set_verifyssl(self, verifyssl):
         self._verifyssl = verifyssl
         self.verifyssl_changed.emit()
         log.debug("Liveimg ssl verification is set to '%s'", self._verifyssl)
+
+    @property
+    def required_space(self):
+        """Get space required for the source image.
+
+        :rtype: int
+        """
+        return self._required_space
+
+    def set_required_space(self, required_space):
+        """Set space required for the source image."""
+        self._required_space = required_space
+        self.required_space_changed.emit()
+        log.debug("Space required for source image is set to '%s'", self._required_space)
+
+    @property
+    def requests_session(self):
+        """Get requests session."""
+        # FIXME: share in Payload module?
+        if not self._requests_session:
+            self._requests_session = requests_session()
+        return self._requests_session
+
+    def update_kernel_version_list(self):
```

Comment:
I wasn't quite sure about kernel_version_list interface, I'd leave it to Jirka / follow-up patches.
It is updated at different stages (setup, pre-install) for LiveOS and LiveImage so I tried to suggest the need of doing the update the right time by client (Payload? Handler? as part of the hook?)

Reply:
This is happening always after mount because we will have the list even when someone calls teardown. Not sure if we need that, I'll look on that in the future work.

Reply:
Ok, no problem.